app/main.py

```diff
@@ -1 +1,67 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        if skill not in self.skills:
+            self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = [
+            "JavaScript",
+            "HTML",
+            "CSS",
+        ]
```

Comment:
You have to supplement `self.skills` instead of rewrite this value
```suggestion
        self.skills += [
            "JavaScript",
            "HTML",
            "CSS",
        ]
```